parser_tool/static/js/src/case_distribution.js

```diff
@@ -0,0 +1,302 @@
+                            ]);
+                            return cfg.w / 2 * (1 - (Math.max(j.value, 0) / cfg.maxValue) * cfg.factor * Math.sin(i * cfg.radians / total));
+                        })
+                        .attr("cy", function (j, i) {
+                            return cfg.h / 2 * (1 - (Math.max(j.value, 0) / cfg.maxValue) * cfg.factor * Math.cos(i * cfg.radians / total));
+                        })
+                        .attr("data-id", function (j) { return j.axis })
+                        .style("fill", cfg.color(series)).style("fill-opacity", .9)
+                        .on('mouseover', function (d) {
+                            newX = parseFloat(d3.select(this).attr('cx')) - 10;
+                            newY = parseFloat(d3.select(this).attr('cy')) - 5;
+
+                            tooltip
+                                .attr('x', newX)
+                                .attr('y', newY)
+                                .text(Format(d.value))
+                                .transition(200)
+                                .style('opacity', 1);
+
+                            z = "polygon." + d3.select(this).attr("class");
+                            g.selectAll("polygon")
+                                .transition(200)
+                                .style("fill-opacity", 0.1);
+                            g.selectAll(z)
+                                .transition(200)
+                                .style("fill-opacity", .7);
+                        })
+                        .on('mouseout', function () {
+                            tooltip
+                                .transition(200)
+                                .style('opacity', 0);
+                            g.selectAll("polygon")
+                                .transition(200)
+                                .style("fill-opacity", cfg.opacityArea);
+                        })
+                        .append("svg:title")
+                        .text(function (j) { return Math.max(j.value, 0) });
+
+                    series++;
+                });
+                //Tooltip
+                tooltip = g.append('text')
+                    .style('opacity', 0)
+                    .style('font-family', 'sans-serif')
+                    .style('font-size', '20px');
+            }
+        };
+        return RadarChart
+    }
+    async function make_chart(data, lemma_lookup) {
+        $("#error").hide();
+        var input_text = $("#contentinput").val().trim();
+        var index = lemma_lookup[input_text];
+
+        if (index == undefined) {
+            $("#output").hide();
+            $("#error").show();
+            return false;
+        }
+
+        var radar_data = data[index];
+        $("#lemma").text(input_text);
+        $("#output").show();
+
+        get_radar_config().then(function (result) {
```

Comment:
In line with removing the `async` keyword from the `get_radar_config` definition, you should be able to treat this as a regular function call:

```javascript
var RadarChart = get_radar_config();
...
```